### DISCUSSION TITLE
Special Request for help

### DISCUSSION THREAD

**Discussion by JEBurroughs** [mriscoc/Ender3V2S1#788] [General]
Hey, I just don't seem to compute when it comes to the write your own bin file, so I was wondering if one of you nice smart guys could find the time to create a special configuration for me. I have 2 - 4.2.7 Ender 3 boards. They run smooth as silk with the October files that were posted here, however for some reason, with the December and now this new March release, my Z axis is not raising correctly. If I start the print, I have to keep raising the Z via the "Tune/ Probe Z Offset" adjustment. About every 6 or 8 mm in lift, I have to give it about +0.05 mm increase so that the flow doesn't smear and drag. If I just let it go, you will eventually hear the nozzle scraping the print.  If you lift the Z then it's good for another 6 or 8 mm's.

SO, What I am wanting, is if someone could take the "Ender3V2-427-BLTUBL-20221002.bin"  configuration and add the feature where the Mesh Viewer during the "Auto Build Mesh" action shows the digits on the screen like the 2 latest build do. I just like seeing the numbers while it's doing the Auto Mesh. I would also like to add the "Live Movement" tick box for the Axis Movements.

It would just be adding those 2 features to the October Build Ender3V2-427-BLTUBL-20221002.bin for me. I understand it's time consuming so I appreciate your help and totally understand if you tell me to get lost. I wish the new builds would would work for me, and more so I wonder why they don't seem to work in my 2 boards.

Thanks, JB

**Comment by JEBurroughs** [comment]
Just wanted to add, I did check the Z steps and they are spot on at 100mm's. Example I am printing a desk clamp now. I started the print at -0.60 mm on the Z offset. The print is at 87% or 32.12 mm high, and the Z offset is at -0.43. I have had to raise it +0.17mm during the build so far. This happens with December and March, but not with October. I have re-flashed both boards a few different times, and the results are consistent. If I were to just let it go, the top layer would be scraped all to hell.

**Comment by mriscoc** [comment]
If the last two versions don't work for you, maybe it is a configuration mistake, remember that versions use UBL not ABL. You could have a bad mesh leveling saved in the UBL area of EEPROM memory.

**Comment by JEBurroughs** [comment]
10-4, I Have a BLTouch on both, and I am using the UBL. I save the Mesh to slot 0,  and I call it up in the PrusaSlicer GCode start along with the bed tilt.

G90 ; use absolute coordinates
M83 ; extruder relative mode
M140 S{first_layer_bed_temperature[0]} ; set final bed temp
M104 S150 ; set temporary nozzle temp to prevent oozing during homing
G4 S10 ; allow partial nozzle warmup
G28 ; home all axis
G29 A ; Activates Bed Leveling  Old settings for load mesh stored are  M420 S1
G29 L0 ; Loads First Saved Mesh
G29 J3 ; Probes the bed for mesh tilt
G1 Z50 F240
G1 X2 Y10 F3000
M104 S{first_layer_temperature[0]} ; set final nozzle temp
M190 S{first_layer_bed_temperature[0]} ; wait for bed temp to stabilize
M109 S{first_layer_temperature[0]} ; wait for nozzle temp to stabilize
G1 Z0.28 F240
G92 E0
G1 Y140 E10 F1500 ; prime the nozzle
G1 X2.3 F5000
G92 E0
G1 Y10 E10 F1200 ; prime the nozzle
G92 E0


So wont re-flashing erase what was in the memory previously? I guess what is strange is I can have one of the newer bin's in and it does the weird Z issue, then I re-flash to the old October and she's back printing nice again, re-flash to a newer version and back to the old Z issue. It's just so crazy. 
Now let me throw this in the mix. The Boards are originally, Pro boards not V2's, but if I understand correctly thy are the same.
